kong/plugins/proxy-cache/cache_key.lua

```diff
@@ -108,8 +109,8 @@ function _M.build_cache_key(consumer_id, route_id, method, uri,
   local params_digest  = params_key(params_table, conf)
   local headers_digest = headers_key(headers_table, conf)
 
-  return md5(fmt("%s|%s|%s|%s|%s", prefix_digest, method, uri, params_digest,
-                                   headers_digest))
+  return sha256_hex(fmt("%s|%s|%s|%s|%s", prefix_digest, method, uri,
+                          params_digest, headers_digest))
```

Comment:
style: should have the same alignment as before.

Reply:
done, thank you